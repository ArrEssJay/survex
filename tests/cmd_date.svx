; valid dates:
*begin
*date 70.01.01
*date 70.01
*date 70.02
*date 70
*date 99.12.31
*date 1970-2000
*date 1970.01.01
*date 1970.01
*date 1970.02
*date 1970
*date 1970-2000
*date 1975.01.31
*date 1975.02.28
*date 1975.03.31
*date 1975.04.30
*date 1975.05.31
*date 1975.06.30
*date 1975.07.31
*date 1975.08.31
*date 1975.09.30
*date 1975.10.31
*date 1975.11.30
*date 1975.12.31
*date 1976.02.29
*date 1976.03.31
*date 1976.04.30
*date 1976.05.31
*date 1976.06.30
*date 1976.07.31
*date 1976.08.31
*date 1976.09.30
*date 1976.10.31
*date 1976.11.30
*date 1976.12.31
*date 2000.01.31
*date 2000.02.28
*date 2000.03.31
*date 2000.04.30
*date 2000.05.31
*date 2000.06.30
*date 2000.07.31
*date 2000.08.31
*date 2000.09.30
*date 2000.10.31
*date 2000.11.30
*date 2000.12.31
*date 2000.01.31
; these should generate one "date in future" warning each (unless it's now
; 2037!)
*date 2037.12.31
*date 2037.06
*date 2037
*fix 1 0 0 0
1 2 1.0 020 -03
*end
