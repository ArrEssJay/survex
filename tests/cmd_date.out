./cmd_date.svx:4:7: warning: Assuming 2 digit year is 1900
 *date 00.01.01
       ^~
./cmd_date.svx:5:7: warning: Assuming 2 digit year is 1900
 *date 00.01
       ^~
./cmd_date.svx:6:7: warning: Assuming 2 digit year is 1900
 *date 00.02
       ^~
./cmd_date.svx:7:7: warning: Assuming 2 digit year is 1900
 *date 00
       ^~
./cmd_date.svx:14:7: warning: Assuming 2 digit year is 1902
 *date 02.01.01
       ^~
./cmd_date.svx:15:7: warning: Assuming 2 digit year is 1902
 *date 02.01
       ^~
./cmd_date.svx:16:7: warning: Assuming 2 digit year is 1902
 *date 02.02
       ^~
./cmd_date.svx:17:7: warning: Assuming 2 digit year is 1902
 *date 02
       ^~
./cmd_date.svx:24:7: warning: Assuming 2 digit year is 1969
 *date 69.12.31
       ^~
./cmd_date.svx:28:7: warning: Assuming 2 digit year is 1970
 *date 70.01.01
       ^~
./cmd_date.svx:29:7: warning: Assuming 2 digit year is 1970
 *date 70.01
       ^~
./cmd_date.svx:30:7: warning: Assuming 2 digit year is 1970
 *date 70.02
       ^~
./cmd_date.svx:31:7: warning: Assuming 2 digit year is 1970
 *date 70
       ^~
./cmd_date.svx:32:7: warning: Assuming 2 digit year is 1999
 *date 99.12.31
       ^~
./cmd_date.svx:76:12: warning: Date is in the future!
 *date 1900-2078
            ^~~~
./cmd_date.svx:77:7: warning: Date is in the future!
 *date 2078.12.31
       ^~~~~~~~~~
./cmd_date.svx:78:7: warning: Date is in the future!
 *date 2078.06
       ^~~~~~~
./cmd_date.svx:79:7: warning: Date is in the future!
 *date 2078
       ^~~~
./cmd_date.svx:80:18: warning: Date is in the future!
 *date 2016.08.25-2078.12.31 ; long survey trip!
                  ^~~~~~~~~~
./cmd_date.svx:82:7: warning: Date is in the future!
 *date 2078.12.01-2078.12.31
       ^~~~~~~~~~
./cmd_date.svx:82:18: warning: Date is in the future!
 *date 2078.12.01-2078.12.31
                  ^~~~~~~~~~

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 2 survey stations, joined by 1 leg.
There are 0 loops.
Total length of survey legs =    1.00m (   1.00m adjusted)
Total plan length of survey legs =    1.00m
Total vertical length of survey legs =    0.05m
Vertical range = 0.05m (from 1 at 0.00m to 2 at -0.05m)
North-South range = 0.94m (from 2 at 0.94m to 1 at 0.00m)
East-West range = 0.34m (from 2 at 0.34m to 1 at 0.00m)
   2 1-nodes.
There were 21 warning(s).
